collect_app/src/main/java/org/odk/collect/android/application/initialization/LocationInitializer.kt

```diff
@@ -0,0 +1,14 @@
+package org.odk.collect.android.application.initialization
+
+import org.odk.collect.android.R
+import org.odk.collect.location.tracker.ForegroundServiceLocationTracker
+
+/**
+ * Initializes the [org.odk.collect.location] framework
+ */
+class LocationInitializer : Initializer {
+
+    override fun initialize() {
+        ForegroundServiceLocationTracker.notificationIcon = R.drawable.ic_notes_white
```

Comment:
Can't we just use this icon statically in `ForegroundServiceLocationTracker` like we do in case of all the other notifications? Do we need to initialize it like that?

Reply:
I guess my thinking was that the icon is "Collect" specific and shouldn't really be in the `location` module

Reply:
I see then that icon could be in `androidshared` or even we could have a separate module for drawables like we have for strings as it is possible to use the same drawables in different modules (rather in the future). You are a master of abstractions and I usually like the way you think but here it's too much for me I guess especially now as you want to add this to v2022.1 it should be a small fix without touching all those initializes (you can do that in a separate pr).

Reply:
> You are a master of abstractions and I usually like the way you think

🙏

> ...but here it's too much for me I guess especially now as you want to add this to v2022.1

Yeah I do see what you're saying. I'm trying to adhere to a self enforced rule that we should be able to pull the module out and use it in another application but needing to do initialization like this is a pretty clunky way of making that work. We can probably do something nicer with a shared interface for grabbing the "notification icon" from the `Application` but for now let's just move the icon to `location`. We're using a different icon for `AudioRecorderService` which I don't think makes sense - really all the app notifications should use a consistent icon. I can create an issue for harmonizing that and we can think about a more "elegant" way to configure the icon then.

Reply:
Ok